### PR TITLE
Added iOS 12 support.

### PR DESCRIPTION
**Pull request by donbytyqi**

### CODE REVIEW

File: Tweak.xm [hunk 1]
```diff
@@ -1,69 +1,38 @@
...
+      snoopImageView.tag = 69;
+      snoopImageView.contentMode = UIViewContentModeScaleAspectFill;
+
+      if (![snoopImageView isDescendantOfView: v.superview.superview.superview]) {
+        [v.superview.superview.superview insertSubview: snoopImageView atIndex: 0];
+      }
+    } else {
+        // directly getting the view with the tag id doess not seem to work, it works for now, i'll look into this later
+        for(UIView *view in v.superview.superview.superview.subviews) {
+            if(view.tag == 69) {
```
Comment (codyd51) [inline Tweak.xm]: Can we make the tag here less likely to collide with another view?

```suggestion
#define kOverlayViewTag someUniqueNumber
...
            if(view.tag == kOverlayViewTag) {
```

**Comment by codyd51** [comment]
Looks good, thanks a lot for updating this!

File: Tweak.xm (hunk 1 above)
Comment (donbytyqi) [reply Tweak.xm]: Yeah sorry forgot to change it

**Comment by donbytyqi** [comment]
> Looks good, thanks a lot for updating this!

No problem!

**Comment by codyd51** [comment]
@uroboro pointed out that this breaks support for pre-iOS 12 devices. This should be added back in at some point.

**Pull request merged**